assemblies/assembly-designing-business-processes.adoc

```diff
@@ -101,8 +100,8 @@ include::{enterprise-dir}/admin-and-config/manual-cleanup-con.adoc[leveloffset=+
 include::{enterprise-dir}/admin-and-config/clean-log-proc.adoc[leveloffset=+2]
 
 include::{jbpm-dir}/BPMN2/process-definitions-and-instances-con.adoc[leveloffset=+1]
-include::{enterprise-dir}/processes/starting-process-instance-from-definition-proc.adoc[leveloffset=+2]
-include::{enterprise-dir}/processes/starting-process-instance-from-instances-proc.adoc[leveloffset=+2]
+include::{jbpm-dir}/BPMN2/starting-process-instance-from-definition-proc.adoc[leveloffset=+2]
```

Comment:
just one concern why rename processes to BPMN2?

Reply:
I moved the files from the **processes** folder to the **BPMN2** folder that's it. 

The **processes** folder was only created in the enterprise document directory.

Reply:
I just think name processes better than BPMN2.
Others look like ok for me.